### ISSUE TITLE
Customer information missing after pulling in from View Walkin 

### ISSUE THREAD

**Issue by wirelessrepairzone** [Repair-DeskPOS/RepairDesk-Bugs#25]
A large number of times now (maybe 1 out of 5) we have customers fill out the Walk In from our website, then from POS we hit View Walk in confirm the information (The customer info does show up), update the price and other items then checkout. Normally we then hit Checkout then confirm without entering a payment as the customer pays when they pickup the item. The label that prints (both the recept and ticket label) show walk in with no customer information. 

There is also no customer information recorded on the ticket or invoice. 

If we use the same procedure without the Checkin and enter everything directly in POS the system works without fail. 

In our system   T-1461, T-1473, T-1523 are examples of this

**Comment by imiamran** [comment]
i am unable to reproduce @HafizAnas can you help me to trace the issue please .

**Comment by wirelessrepairzone** [comment]
im not sure, i happoned again yesterday  the customer fill out the
information at http://wirelessrepairzone.com/checkin.aspx  then we pull up
on POS and verifyall the customer info is filled in. we finish the order
adding the repair then hit checkout. Normally from the checkout screen we
simply hit Confirm without entering any payments (as they pay at pickup)
it then prints them a receipt that we give them to show they dropped off
the item and a label for the device.   Yesterday ticket number T-1617 did
it, the info was in the system and showed up on the screen but when the
label and receipt printed it seemed to loss it and revert back to Walkin
Customer. We did  not notice as we handed the receipt and the customer
left. They went to grab the sticker and seen it was for Walkin so we had
someones phone with no name or contact information to reach them.   It does
not happon each time, i have not seen a trigger that i can tell you but it
happons one out of 10 at least
